docs/deployment/openshift/custom-namespace.md

```diff
@@ -0,0 +1,24 @@
+# Deploying the Node Sensor to a custom Namespace
```

Comment:
One thing to be aware of is that often it is easy for an SE/Cx to interact with a single doc vs multiple docs... especially when passing through email also the operator links here for documentation directly.... hence (while it may seem less than ideal) everything was done per single doc. If the docs should stay separated, having the docs that are broken out need and probably really should be linked and discoverable via the Readme doc.